### PR TITLE
Fix trivy scan failure with testng

### PR DESCRIPTION
**Pull request by TharmiganK**
## Purpose

> $Subject

Fixes : https://github.com/ballerina-platform/ballerina-standard-library/issues/3793

## Examples

N/A

## Checklist
- [ ] <s>Linked to an issue</s>
- [ ] <s>Updated the changelog</s>
- [ ] <s>Added tests</s>
- [ ] <s>Updated the spec</s>
- [ ] <s>Checked native-image compatibility</s>

### CODE REVIEW

File: build-config/resources/BallerinaTest.toml [hunk 1]
```diff
@@ -4,4 +4,9 @@ name = "http_tests"
 version = "@toml.version@"
 
 [[platform.java11.dependency]]
+scope = "testOnly"
```
Comment (dilanSachi) [inline build-config/resources/BallerinaTest.toml:7]: why is `http-native` test only?

Comment (dilanSachi) [reply build-config/resources/BallerinaTest.toml:7]: Ah this is ballerina test module.

File: test-utils/src/main/java/module-info.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+module io.ballerina.stdlib.http.testutils {
+    requires io.netty.buffer;
+    requires io.netty.codec.http;
+    requires io.netty.common;
+    requires org.slf4j;
+    requires org.testng;
+    requires io.ballerina.runtime;
+    requires io.netty.transport;
+    requires io.ballerina.stdlib.http;
+
```
Comment (dilanSachi) [inline test-utils/src/main/java/module-info.java:29]: Lets remove this line

File: build-config/resources/BallerinaTest.toml (hunk 1 above)
Comment (TharmiganK) [reply build-config/resources/BallerinaTest.toml:7]: This is for the ballerina http test module which only have ballerina tests

File: test-utils/src/main/java/module-info.java (hunk 2 above)
Comment (TharmiganK) [inline test-utils/src/main/java/module-info.java:29]:
```suggestion
```

Comment (TharmiganK) [reply test-utils/src/main/java/module-info.java]: Fixed via [026e41b](https://github.com/ballerina-platform/module-ballerina-http/pull/1354/commits/026e41b61cf4e0a745cf14b2903f388633a30a0d)

**Review by dilanSachi — APPROVED** [review]
LGTM

**Pull request merged**